src/components/WheelPicker/wheelPicker.api.json

```diff
@@ -2,6 +2,7 @@
   "name": "WheelPicker",
   "category": "form",
   "description": "A customizable WheelPicker component",
+  "note": "When using Android by default the FlatList will have 'maxToRenderPerBatch' prop set to items.<br/>length to solve FlatList bug on Android, you can override it by passing your own 'flatListProps' with 'maxToRenderPerBatch' prop.<br/>See the RN FlatList issue for more info: https://github.com/facebook/react-native/issues/15990",
```

Comment:
I did not test it but I assume `items.<br/>length` will look strange, it should probably be in the same line and preferably with code look (if possible in the `note`): `items.length`

Reply:
Fixed.
![Screen Shot 2023-05-01 at 16 29 13](https://user-images.githubusercontent.com/12716981/235458380-5a274a83-f2a6-4af0-9011-4786d7436e4d.png)

Reply:
Fixed, checked.
Wrapped with `<code>Text...</code>`